src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Represents a task.
+ */
+public class Task {
+    /** Description of task.*/
+    protected String description;
+    /** Boolean flag of whether task is done.*/
+    protected boolean isDone;
+    /** Type of Task.*/
+    protected String typeOfTask;
+
+    /** Task object Constructor.*/
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Retrieves status icon for task.
+     * */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Consider using a constant variable named "TICK_SYMBOL" for "\u2713" otherwise it is a magic value in the code (same for "\u2718").